docs/00 - Stellar Anchor Platform.md

```diff
@@ -0,0 +1,98 @@
+<div>
+<img alt="Stellar" src="https://github.com/stellar/.github/raw/master/stellar-logo.png" width="558" />
+<br/>
+<strong>Creating equitable access to the global financial system</strong>
+</div>
+
+# Stellar Anchor Platform
+
+The Anchor Platform is a web application whose goal is to facilitate Anchor
+integration with the Stellar network in an interoperable way, compiant with the
+[SEP (Stellar Ecosystem Protocols)](https://github.com/stellar/stellar-protocol/tree/master/ecosystem).
+
+The Anchor Platform will be exposing public endpoints and APIs that provide
+functionalities described in the [SEPs] while abstracting most of the business
+logic necessarry to enable interoperability with the Stellar network.
+
+This way, Anchors willing to integrate with Stellar can use the Anchor Platform
+to speed up their development process, allowing them to focus mainly on the
+business logic that's specific to their businesses and use cases.
```

Comment:
I think content here is good, but I think we can be a bit stronger with the language used to imply that this is the best way to run an anchor.

```suggestion
The Anchor Platform is the easiest and fastest way to deploy a [SEP-compatible](https://github.com/stellar/stellar-protocol/tree/master/ecosystem) anchor service. 

It implements a the majority of standardized API (SEP) endpoints that wallets, exchanges, and other applications use, and provides a set of backend HTTPS APIs & callbacks for the anchor to integrate with for specifying fees, exchange rates, and off-chain transaction status updates.

The goal of the Anchor Platform is to abstract all Stellar-specific functionality and requirements for running an anchor, allowing businesses to focus on the core business logic necessary to provide these services.
```